kafka-panache-quickstart/README.md

```diff
@@ -0,0 +1,37 @@
+Quarkus Kafka/Panache Quickstart
+================================
+
+This project illustrates how you can interact with Apache Kafka using MicroProfile Reactive Messaging and Hibernate with Panache.
+
+## Infrastructure
+
+You need a PostGreSQL instance and a Kafka broker.
```

Comment:
Is that the right capitalization for Postgres?

Reply:
Almost :-) PostgreSQL